src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -295,12 +295,18 @@ TEST_F(ParseBigManifestTest, ManifestProtoClaimsTest) {
 }
 
 static std::string kExpectedCheckStrings[] = {
-    R"(checkHasTag("NamedR.PS1#0.in_handle.field1", "tag2").)",
-    R"(checkHasTag("NamedR.PS1#1.in_handle.field1", "tag2").)",
-    R"(checkHasTag("NamedR.PS2#2.in_handle.field1", "tag4").)",
-    R"(checkHasTag("GENERATED_RECIPE_NAME0.PS1#0.in_handle.field1", "tag2").)",
-    R"(checkHasTag("GENERATED_RECIPE_NAME0.PS2#1.in_handle.field1", "tag4").)",
-    R"(checkHasTag("GENERATED_RECIPE_NAME0.PS2#2.in_handle.field1", "tag4").)",
+    R"(checkHasTag("NamedR.PS1#0.in_handle.field1", "tag2") :-)"
+    R"( mayHaveTag("NamedR.PS1#0.in_handle.field1", "tag2").)",
```

Comment:
Rather than two raw strings which are concatenated together, it would be more readable to have one raw string that exceeds the 80 character limit. Going over the 80 character limit is allowed for raw strings.

Reply:
Done.